external/dynamo/generalized/frontend.py

```
@@ -0,0 +1,446 @@
+        self.runtime = runtime
+        self.processor_client = None
+        self.app = None
+        self.tokenizers: dict[str, AutoTokenizer] = {}
+        # Regex to find one or more JSON objects optionally separated by semicolons
+
+        # Throughput (requests/sec) tracking
+        self._tps_lock = asyncio.Lock()
+        self._tps_count = 0
+        try:
+            self._tps_interval = float(os.environ.get("FRONTEND_TPS_INTERVAL", "5"))
+        except Exception:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)